### PR TITLE
Added specific contributor-guidance

### PR DESCRIPTION
**Pull request by benjagm**
**GitHub Issue:** # / NA

**Summary**:

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

Yes/No
[JUSTIFICATION]

### CODE REVIEW

File: programs/mentoring/gsoc/README.md [hunk 1]
```diff
@@ -51,5 +51,5 @@ Got a problem? Reach out to mentors or the community for assistance. Remember, m
 
 ## 🏗 Contributing Guidelines
 
-Please, check-out our [Contribution guidelines](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md)
+Please, check-out the [GSoC Contributior guidelines](CONTRIBUTOR-GUIDANCE.md)
```
Comment (notEthan) [inline programs/mentoring/gsoc/README.md:54]: /Contributior/Contributor/

File: programs/mentoring/gsoc/gsoc-2024.md [hunk 2]
```diff
@@ -81,4 +81,4 @@ Got a problem? Reach out to mentors or the community for assistance. Remember, m
 
 ## Contributing Guidelines
 
-Please, check-out our [Contribution guidelines](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md)
+Please, check-out the [GSoC Contributior guidelines](CONTRIBUTOR-GUIDANCE.md)
```
Comment (notEthan) [inline programs/mentoring/gsoc/gsoc-2024.md:84]: same

File: programs/mentoring/gsoc/README.md (hunk 1 above)
Comment (benjagm) [reply programs/mentoring/gsoc/README.md:54]: Thanks @notEthan !

File: programs/mentoring/gsoc/gsoc-2024.md (hunk 2 above)
Comment (benjagm) [reply programs/mentoring/gsoc/gsoc-2024.md:84]: Thanks for the heads up!

**Pull request merged**